routing/missioncontrol_store.go

```diff
@@ -64,6 +61,11 @@ func newMissionControlStore(db kvdb.Backend, maxRecords int) (*missionControlSto
 		}
 
 		return nil
+	}, func() {
+		store = &missionControlStore{
```

Comment:
there was another similar case like this (`s.cache` I think) where we did set up the struct with the `db` field set outside the reset closure. Doesn't matter much, but perhaps makes sense to be consistent style wise.